pulp_python/app/viewsets.py

```diff
@@ -30,6 +31,15 @@
     "sdist": pkginfo.SDist,
 }
 
+class PythonPackageContentFilter(filterset.FilterSet):
+    class Meta:
+        model = python_models.PythonPackageContent
+        fields = {
+            'type': ['exact', 'in'],
+            'author': ['exact', 'in'],
+            'packagetype': ['exact', 'in'],
+            'filename': ['exact', 'in', 'contains'],
```

Comment:
Added a contains filter here- this is useful for users trying to find all distributions of a project version:
`http://pulp3.dev:8000/pulp/api/v3/content/python/packages/?filename__contains=pulpcore-3.0.0`